src/components/badge/index.js

```diff
@@ -22,11 +22,9 @@ class Badge extends Component {
      * @return {VirtualComponent} The Badge component.
      */
     render($$) {
-        const { text } = this.props;
-
         return $$('span')
             .addClass(styles.badge)
-            .setTextContent(text);
+            .setTextContent(this.props.text);
```

Comment:
This is functionally the same as the code above, it is just not destructuring the props object into new variables. This way is fine, just noting that it is the same thing :)